src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,80 @@
+package duke.storage;
+
+import duke.dukeexceptions.DukeExceptions;
+import duke.task.Task;
+import duke.tasklist.TaskList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * The Storage object acts as an interface between Duke and the filename for database management.
+ */
+public class Storage{
+    /** The file which stores the tasks in the task list. */
+    private File file;
+
+    /**
+     * Creates a new Storage object with the filename to store and retrieves the data in the tasks list.
+     *
+     * @param filename THe file where the tasks in the tasks list is retrieved and stored.
+     */
+    public Storage(String filename) {
+        this.file = new File(filename);
+    }
+
+    /**
+     * Gets all the tasks in file and creates a new task list based on the data retrieved.
+     *
+     * @return Task List with all the tasks retrieved from the file filename.
+     * @throws FileNotFoundException If the file filename could not be found in duke.
+     */
+    public TaskList getData() throws FileNotFoundException {
+        Scanner fileInput = new Scanner(file);
+        TaskList taskList = new TaskList();
+
+        // Gets the data from the file and creates new task into the task list.
+        while (fileInput.hasNextLine()) {
+            // Gets the properties for a task.
+            String type = fileInput.nextLine();
+            Boolean done = Boolean.parseBoolean(fileInput.nextLine());
```

Comment:
Perhaps using a prefix such as is to the variable name such as IsDone like @hieunm1821 mentioned would make it sound more like booleans!